src/robusta/core/model/nodes.py

```diff
@@ -0,0 +1,94 @@
+from pydantic.main import BaseModel
+from typing import List, Dict
+
+
+class NodeInfo(BaseModel):
```

Comment:
This file is only used in the Robusta sink so I don't think it belongs in `core/model/`. Best to reduce the scope and simplify the cross module dependencies.

Reply:
I think it's fine to have it here.
The core/model is the place to have the data model classes.
It might be used somewhere else as well.
And the service info is already defined there too

Reply:
K